cluster.go

```diff
@@ -89,15 +89,20 @@ func (c *cluster) Close() error {
 	return anError
 }
 
-func (c *cluster) getConsumerGroup(consumerGroupName string) (sarama.ConsumerGroup, error) {
+func (c *cluster) getConsumerGroup(consumerGroupName string, config *sarama.Config) (sarama.ConsumerGroup, error) {
```

Comment:
I think it would be clearer and easier to read if in this method you expect to have always a config and then no need of pointer param.

Then you always call NewConsumerGroup with this config.

And you prepare in initialize method the right config to use